src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -935,24 +832,8 @@ private async Task<DialogResult> GetFileListDialog(IEnumerable<string> source, s
             return await dialogService.ShowDialogAsync(dialogViewModel);
         }
 
-        private async Task<List<Win32Process>> WhoIsLockingAsync(IEnumerable<string> filesToCheck)
-        {
-            var connection = await AppServiceConnectionHelper.Instance;
-            if (connection != null)
-            {
-                var (status, response) = await connection.SendMessageForResponseAsync(new ValueSet()
-                {
-                    { "Arguments", "FileOperation" },
-                    { "fileop", "CheckFileInUse" },
-                    { "filepath", string.Join('|', filesToCheck) }
-                });
-                if (status == AppServiceResponseStatus.Success && response.ContainsKey("Processes"))
-                {
-                    return JsonSerializer.Deserialize<List<Win32Process>>(response["Processes"]);
-                }
-            }
-            return null;
-        }
+        private List<Win32Process> WhoIsLocking(IEnumerable<string> filesToCheck)
+            => (List<Win32Process>?)FileOperationsHelpers.CheckFileInUse(string.Join('|', filesToCheck)) ?? new List<Win32Process>();
```

Comment:
Given that the interop between Files.App and Files.FullTrust is no longer needed, we shouldn't serialize paths to a string anymore. Instead we should pass the `string[]` for paths directly.